Update docs references to a/the Prefect server (#9977)
PrefectHQ/prefect: docs/host/index.md

Shut down the Prefect server with <kdb> ctrl </kbd> + <kdb> c </kbd> in the terminal.

### Differences between a Prefect server and Prefect Cloud

> The self-hosted Prefect server and Prefect Cloud share a base of features. Prefect Cloud also includes the following features that you can read about in the [Cloud](/cloud/) section of the docs. 

Reviewer: ```suggestion
The self-hosted Prefect server and Prefect Cloud share a common set of features. Prefect Cloud also includes the following features:
```
